src/components/ReportWelcomeText.js

```diff
@@ -117,11 +125,13 @@ const ReportWelcomeText = (props) => {
                     <Text style={styles.textAlignCenter}>
                         {props.translate('reportActionsView.beginningOfChatHistory')}
                     </Text>
-                    {_.map(displayNamesWithTooltips, ({displayName, pronouns}, index) => (
+                    {_.map(displayNamesWithTooltips, ({displayName, pronouns, login}, index) => (
```

Comment:
NAB: I think it'll be a nice touch to add a Tooltip for the display names.

```
<Tooltip text={tooltip}>
    <Pressable ...
</Tooltip>
```

<img width="230" alt="image" src="https://user-images.githubusercontent.com/29673073/163885428-fdca3461-6339-4948-ad81-1169a4c60bc2.png">

Reply:
Done.